frontend/app/src/composables/message-handling.ts

```diff
@@ -77,7 +83,8 @@ export const useMessageHandling = () => {
               'notification_messages.premium.inactive.network_problem_message'
             ),
         display: true,
-        severity: Severity.ERROR
+        severity: Severity.ERROR,
+        type: SocketMessageType.PREMIUM_STATUS_UPDATE
```

Comment:
Question, why not make a priority enum instead of exposing the type? 

it could be something like: 

```typescript
enum Priority {
  BULK,
  NORMAL,
  HIGH,
  ACTION
}
```

And since it is an enum we can just order by the value then?

Reply:
done